devito/core/arm.py

```diff
@@ -1,6 +1,6 @@
 from devito.core.cpu import Cpu64AdvOperator
 from devito.passes.iet import (CTarget, OmpTarget, mpiize, optimize_halospots,
-                               hoist_prodders, relax_incr_dimensions)
+                               hoist_prodders, linearize, relax_incr_dimensions)
```

Comment:
Should this be a bit more explicit, like `linearize_index`

Reply:
nope. If you look into `linearize`, you'll see it consists of several sub-passes, among which what you call `linearize_index`. IOW, `linearize_index` is just one of the steps to implement linearization